### PR TITLE
SDCB-9999 [Appium 2.0] Python Server Side

### PR DESCRIPTION
**Pull request by eryk-tkaczewski**

### CODE REVIEW

**Review by michal-przytarski — COMMENTED** [review]
Some not changes related problems with this sample:
 - correct apps should be linked in `README.md`
 - in `run-test_ios.sh` there is this code:

```sh
echo "Installing pip"
curl https://bootstrap.pypa.io/get-pip.py | python - --user
``` 

It doesn't work: `ERROR: This script does not work on Python 2.7 The minimum supported Python version is 3.7. Please use https://bootstrap.pypa.io/pip/2.7/get-pip.py instead.`

- Even with this error sample works correctly, so I suggest removing code for installing pip, as its already installed. 
- Exporting PATH for `pip2` and the calling `pip3` is not the best way to to that. Also, as said above, exporting that path is probably unnecessary.
- `requirements.txt` doesn't need to be 755. 444 is just enough for it.
- similar for Android
- it doesn't work on the new Android versions: https://cloud.bitbar.com/#testing/device-session/207265745/210142503/10825775 <- tested on Appium1, because I'm not sure how to use Appium2 in SS tests

To sum this all up: code of this sample is wild, but I believe that `README.md` and `run-test.sh` should be kept in a "good" shape, so please try to fix those thing 😄

File: samples/testing-frameworks/appium/server-side/python/BitBarAppiumTest.py [hunk 1]
```diff
@@ -95,24 +95,22 @@ def set_screenshot_dir(self, screenshot_dir):
     def get_desired_capabilities(self):
         desired_caps = {
```
[the remark this answers is not in the document]
Comment (eryk-tkaczewski) [reply samples/testing-frameworks/appium/server-side/python/BitBarAppiumTest.py]: Done

File: samples/testing-frameworks/appium/server-side/python/README.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ This folder contains a sample Appium server side test case for Android
 and iOS environments. To read more about server and client Appium runs
 in BitBar Cloud, check our online documentation from `https://support.smartbear.com/bitbar/docs/testing-with-bitbar/automated-testing/appium/index.html`.
 
+## Apps
+
+You will find the necessary applications in:
+Android: `test-samples/apps/android/testdroid-sample-app.apk`
```
Comment (michal-przytarski) [inline samples/testing-frameworks/appium/server-side/python/README.md:10]: Wrong app - should be `bitbar-sample-app.apk`

File: samples/testing-frameworks/appium/server-side/python/README.md [hunk 3]
```diff
@@ -4,6 +4,12 @@ This folder contains a sample Appium server side test case for Android
...
+iOS: `test-samples/apps/ios/bitbar-ios-sample.ipa`
```
Comment (michal-przytarski) [inline samples/testing-frameworks/appium/server-side/python/README.md:11]: I believe you wanted to list those apps. To do that please add `-` as unordered list. Linking apps with e.g. `[bitbar-sample-app.apk](../../../../../apps/android/bitbar-sample-app.apk)` may be more user friendly

File: samples/testing-frameworks/appium/server-side/python/run-tests_android.sh [hunk 4]
```diff
@@ -11,7 +11,7 @@ echo "Extracting tests.zip..."
 unzip tests.zip
 
 echo "Installing pip for python"
-curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+curl https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py
 python get-pip.py
```
Comment (michal-przytarski) [inline samples/testing-frameworks/appium/server-side/python/run-tests_android.sh:15]: > Python 2.7 reached the end of its life on January 1st, 2020

We can get rid of this step. We are using `pip3` anyway

File: samples/testing-frameworks/appium/server-side/python/run-tests_ios.sh [hunk 5]
```diff
@@ -16,7 +16,7 @@ unzip -o tests.zip
 #########################################################
 
 echo "Installing pip"
-curl https://bootstrap.pypa.io/get-pip.py | python - --user
```
Comment (michal-przytarski) [inline samples/testing-frameworks/appium/server-side/python/run-tests_ios.sh:19]: > Python 2.7 reached the end of its life on January 1st, 2020

We can get rid of this step. We are using `pip3` anyway

**Pull request closed without merging**